const ExceptionName = {
	AUTHORIZATION_ERROR: "AuthError",
} as const;

export { ExceptionName };
